src/pyscipopt/scip.pxi

```diff
@@ -305,6 +310,19 @@ cdef class Event:
         """gets type of event"""
         return SCIPeventGetType(self.event)
 
+    def getName(self):
+        """gets name of event"""
+        if not EventNames:
+            self._getEventNames()
+        return EventNames[self.getType()]
+
+    def _getEventNames(self):
+        """gets event names"""
+        for name in dir(PY_SCIP_EVENTTYPE):
+            attr = getattr(PY_SCIP_EVENTTYPE, name)
+            if isinstance(attr, int):
+                EventNames[attr] = name
+        
```

Comment:
Would be great if you could implement the __str__ method too (read here for more https://www.digitalocean.com/community/tutorials/python-str-repr-functions)

Reply:
I added it to the Events, Mo, but couldn't do it for the Stages, as they're not from a class.

Reply:
Perfect!